core/libs/shared/src/api.rs

```diff
@@ -389,6 +389,32 @@ impl Request for UpgradeAccountGooglePlayRequest {
     const ROUTE: &'static str = "/upgrade-account-google-play";
 }
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct UpgradeAccountAppStoreRequest {
+    pub original_transaction_id: String,
```

Comment:
what is this "original" business?

Reply:
Original means the first transaction. That is a way you can identify subscriptions.